tenable/io/v3/was/plugins/api.py

```diff
@@ -38,9 +45,98 @@ def details(self, id: int) -> Dict:
+                    ...      ('field_name_2', 'desc')]
+            filter (tuple, Dict, optional):
+                A nestable filter object detailing how to filter the results
+                down to the desired subset.
+                Examples:
+                    >>> ('or', ('and', ('test', 'oper', '1'),
+                    ...                 ('test', 'oper', '2')
+                    ...             ),
+                    ...     'and', ('test', 'oper', 3)
+                    ... )
+                    >>> {
+                    ...  'or': [{
+                    ...      'and': [{
+                    ...              'value': '1',
+                    ...              'operator': 'oper',
+                    ...              'property': '1'
+                    ...          },
+                    ...          {
+                    ...              'value': '2',
+                    ...              'operator': 'oper',
+                    ...              'property': '2'
+                    ...          }
+                    ...      ]
+                    ...  }],
+                    ...  'and': [{
+                    ...      'value': '3',
+                    ...      'operator': 'oper',
+                    ...      'property': 3
+                    ...  }]
+                    ... }
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth: `tio.v3.definitions.was.plugins()`
+                endpoint
+                to get more details.
+            limit (int, optional):
+                Number of objects to be returned in each request.
+                Default and maximum limit is 200.
+            offset (int, optional):
+                The pagination offset to use when requesting the next page of
+                results.
+            num_pages (int, optional):
+                The total number of pages to request before stopping the
+                iterator.
+            return_resp (bool, optional):
+                If set to true, will override the default behavior to return
+                an iterable and will instead return the results for the
+                specific page of data.
+            return_csv (bool, optional):
+                If set to true, it will return the CSV response or
+                iterable (based on return_resp flag). Iterator returns all
+                rows in text/csv format for each call with row headers.
 
-    def search(self, **kwargs):
-        raise NotImplementedError(
-            'Search and Filter functionality will be updated later.'
-        )
+        Returns:
+            Iterable:
+                The iterable that handles the pagination for the job.
+            requests.Response:
+                If ``return_json`` was set to ``True``, then a response
+                object is instead returned instead of an iterable.
```

Comment:
Can you see, if https://github.com/dhvanilshah-tenable/pyTenable/pull/92#discussion_r808844203 applies here too ?

Reply:
Yes updated the docstring accordingly